### PR TITLE
Agrim/dapi547/langswitcher

### PR DESCRIPTION
**Pull request by agrim-deriv**
Implementing language switcher design for desktop and mobile sidebar view

### CODE REVIEW

File: src/theme/Navbar/MobileSidebar/PrimaryMenu/index.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              {...item}
+              onClick={() => {
+                mobileSidebar.toggle();
+              }}
+            />
+          </ErrorCauseBoundary>
+        ))}
+      </div>
+      <div className='navbar__item navbar__link' onClick={toggleLanguageSidebar}>
+        <LabelPairedGlobeCaptionRegularIcon /> EN
```
Comment (shafin-deriv) [inline src/theme/Navbar/MobileSidebar/PrimaryMenu/index.tsx:61]: @agrim-deriv Why 'EN' is fixed here? Shouldn't it change dynamically based on the chosen language?

Comment (agrim-deriv) [reply src/theme/Navbar/MobileSidebar/PrimaryMenu/index.tsx]: Fixed this @shafin-deriv

File: docusaurus.config.js [hunk 2]
```diff
@@ -9,14 +9,14 @@ const config = {
...
-  onBrokenLinks: 'throw',
+  onBrokenLinks: 'warn',
   onBrokenMarkdownLinks: 'warn',
   favicon: 'img/favicon.png',
   // GitHub pages deployment config.
   // If you aren't using GitHub pages, you don't need these.
   organizationName: 'binary-com', // Usually your GitHub org/user name.
   projectName: 'deriv-api-docs', // Usually your repo name.
-  trailingSlash: false,
+  trailingSlash: true,
```
Comment (shafin-deriv) [inline docusaurus.config.js:19]: @agrim-deriv is these necessary now?

Comment (agrim-deriv) [reply docusaurus.config.js]: No, now it can be removed.

File: docusaurus.config.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const config = {
   // If you aren't using GitHub pages, you don't need these.
   organizationName: 'binary-com', // Usually your GitHub org/user name.
   projectName: 'deriv-api-docs', // Usually your repo name.
-  trailingSlash: false,
+  // trailingSlash: true,
```
Comment (shafin-deriv) [inline docusaurus.config.js:19]: @agrim-deriv can we remove it instead of comment? 😸

**Review by shafin-deriv — APPROVED** [review]

**Review by shafin-deriv — APPROVED** [review]

**Pull request merged**